### DISCUSSION TITLE
Fraction absorbed

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#1175] [Questions & Problems] [answered]
Dear open systems pharmacology team,
I have been working on Rivaroxaban using pksim for a while now and i have been facing a problem , the reported fraction absorbed after i run my simulation is highly under estimated and it is about 0.28 ! And even though this medication has a limited solubility when i change the formulation and dissolution profile , it does not seem to affect the fraction absorbed which does not seem rational also the bioavailibity of Riv is between 50-100% 
How can i increase the fraction absorbed and solve this problem?
Your help is much appreciated

**Comment by tobiasK2001** [comment, marked as answer]
Hi, you might want to check the Intestinal permeability (transcellular) as well. You can make a quick sensitivity analysis with increasing your Reference solubility or the Intestinal permeability (transcellular) by 10 fold and check the impact on FA.
Hope this might help.

Best, Tobias

**Reply by ranamjz** [reply]
> Hi, you might want to check the Intestinal permeability (transcellular) as well. You can make a quick sensitivity analysis with increasing your Reference solubility or the Intestinal permeability (transcellular) by 10 fold and check the impact on FA.
> Hope this might help.
> 
> Best, Tobias

Hello dear tobias thank you for your help the fraction absorbed has increased but pk sim still calculates MRT and half life way off base and highly underestimated
